admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -36,6 +38,7 @@ Always start by making a fresh backup and disabling all 3rd party apps.
    directory to see if they are there. If not, copy them from your old ``apps/``
    directory to your new one. Make sure the directory permissions of your third
    party application directories are the same as for the other ones.
+   ## THIS APPS part IS INSUFFICIENTLY DOCUMENTED (needs better instructions)
```

Comment:
What is missing here? It looks good for me.

Reply:
As a new nextcloud server sysadmin, I found it vague.  The instructions give basic direction, but ideally specific steps could be provided.

Reply:
(for now I think its fine; just made note of it)